Model/User.php

```diff
@@ -190,9 +176,7 @@ public function unserialize($serialized)
             $this->salt,
             $this->usernameCanonical,
             $this->username,
-            $this->expired,
             $this->locked,
-            $this->credentialsExpired,
```

Comment:
This breaks BC in a very bad way: unserializing data from the old version of the bundle would create a corrupted object now (storing things in the wrong properties), and the user is serialized in the session. Until now, we always worked hard on preserving BC here.